honeybee_model_schema/energy/idealair.py

```diff
@@ -8,31 +8,37 @@ class EconomizerType(str, Enum):
     differential_dry_bulb = 'DifferentialDryBulb'
     differential_enthalpy = 'DifferentialEnthalpy'
 
+class IdealAirLimitConfig(str, Enum):
+    autosize = 'AutoSize'
+    no_limit = 'NoLimit'
+    fixed_value = 'FixedValue'
 
-class IdealAirSystem(BaseModel):
-    """ Provides a model for an ideal HVAC system."""
-    type: Enum('IdealAirSystem', {'type': 'IdealAirSystem'})
+class IdealAirLimit(BaseModel):
 
-    heating_limit: Union[float, str] = Schema(
-        'autosize',
+    config: IdealAirLimitConfig = IdealAirLimitConfig.autosize
+
+    value: float = Schema(
+        default=None,
         ge=0
     )
 
-    @validator('heating_limit')
-    def check_string_heating_limit(cls, v):
-        if not isinstance(v ,float) and v != 'autosize' and v != 'NoLimit':
-            raise ValueError( 'This is not a valid entry for heating_limit')
+    @validator('value')
+    def check_config(cls, v, values, **kwargs):
+        if 'value' in values and values['config'] == "FixedValue":
```

Comment:
I'm not sure I like the idea of anothet nested sub-object under the ideal air object. Do you feel what you have here is better than a flatter implementation?

Reply:
Another option is to use float(inf+) instead of "NoLimit" and None instead of "autosize" but this feels somewhat implicit to me.

Reply:
Yeah I know what you mean about adding nested properties to objects. I guess the difficulty here is that I'm not sure what `float(inf+)` looks like in json schema and as you said having `autosize` as a None value seems implicit. Seems like it's a choice between the better of two evils...